SUMMARY.md

```diff
@@ -24,6 +24,10 @@
 * [Tags and placeholders](applications/advanced-concepts/tags-and-placeholders.md)
 * [Multi-engine machine translation](applications/advanced-concepts/multi-engine-machine-translation.md)
 
+## Postediting
```

Comment:
Please note that we have spelled the term "post-editing" in other articles, ie https://www.machinetranslate.org/workflows/post-editing

To avoid repeating the name, this section could be named "Integration"? Maybe within Product?
@bittlingmayer

Reply:
*Integrations* would be fine for now.  I think later we need to split off things like this which are less evergreen, it's more like a tutorial.

Reply:
This is now no more "Post-editing", but "Integrations".